department-of-veterans-affairs/caseflow
# Add guest link and update format of virtual hearings link

Resolves #13936 

### Description
Updated Hearing Details UI based on figma diagram: https://www.figma.com/file/V87TZArfdurCGJiEjQ73ES/Virtual-Hearings?node-id=425%3A15036

### Acceptance Criteria
- [ ] User sees "Scheduling in progress" when conference is being created and emails generated
- [ ] User sees virtual hearing details and a button to copy both guest and host link when the virtual hearing has been scheduled
- [ ] User sees expired when the virtual hearing is past or if it is cancelled

### Testing Plan
1. Select judge "BVAYELLOW": http://localhost:3000/test/users
1. Navigate to hearings:  http://localhost:3000/hearings/schedule
1. Choose a "Video" hearing
1. Click the "Edit hearing details" link
1. Ensure you do not see any virtual hearing details
1. Change from a "Video" hearing to a "Virtual" hearing and ensure you see the "Scheduling in progress" label
1. Once the virtual hearing has been scheduled, confirm that you see details for both the host and guest based on the Figma diagram above.
1. Change the hearing back to "Video" and ensure that you see the expired status based on the Figma diagram

### User Facing Changes

 BEFORE|AFTER
 ---|---
*Previous Virtual Hearing*
![BeforeHearingLinkPrevious](https://user-images.githubusercontent.com/61989526/80509413-6184d500-8947-11ea-85c2-40d8ad5e5b7b.png)|![AfterHearingLinkPrevious](https://user-images.githubusercontent.com/61989526/80509648-b32d5f80-8947-11ea-885b-86248d2a70e6.png)
*Virtual Hearing in progress*
![BeforeHearingLinkInProgress](https://user-images.githubusercontent.com/61989526/80509412-6184d500-8947-11ea-87f4-f86f947d11ec.png)|![AfterHearingLinkInProgress](https://user-images.githubusercontent.com/61989526/80727098-4b0b8480-8ad3-11ea-8c3d-2f1d00cddfa4.png)
*Virtual Hearing Active*
![BeforeHearingLinkActive](https://user-images.githubusercontent.com/61989526/80727006-2a432f00-8ad3-11ea-9688-0535cc7ded85.png)|![AfterHearingLinkActive](https://user-images.githubusercontent.com/61989526/80726930-0ed82400-8ad3-11ea-8c9b-9d6303da6ff3.png)
